### ISSUE TITLE
Phone number not visualized correctly on Safari

### ISSUE THREAD

**Issue by giocalitri** [mitodl/micromasters#2450]
![screen shot 2017-01-27 at 8 40 50 am](https://cloud.githubusercontent.com/assets/1005204/22373083/2bf7624e-e46e-11e6-991c-1ad18ce211e9.png)

**Comment by pdpinch** [comment]
Are you talking about the little head silhouette to the right? I think that's a auto-fill feature of Safari. I don't know if there's a (easy) way to make that behave better.

**Comment by giocalitri** [comment]
No, I am talking about the fact that the number are cut in half

**Comment by aliceriot** [comment]
ew that's yucky

any issues with other fields on safari?

**Comment by giocalitri** [comment]
not sure: I can check later
